lib/codemirror.js

```diff
@@ -3607,6 +3609,12 @@
     var display = cm.display, doc = cm.doc;
     e_preventDefault(e);
 
+    // #3261: make sure, that we're not starting a second selection
+    if (cm.state.selectingText) {
+      cm.state.selectingText(e);
+      return;
```

Comment:
Why not check this in `onMouseDown`? That'd also save the extra condition above (when deciding between a drag and a select).